### PR TITLE
Feat/edit assistant

### PR DESCRIPTION
**Pull request by fontanierh**

### CODE REVIEW

File: front/pages/w/[wId]/builder/assistants/[aId]/index.tsx [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+      user,
+      owner,
+      gaTrackingId: GA_TRACKING_ID,
+      dataSources: allDataSources,
+      agentConfiguration: config,
+    },
+  };
+};
+
+export default function CreateAssistant({
```
Comment (spolu) [inline front/pages/w/[wId]/builder/assistants/[aId]/index.tsx:66]: Edit Assistant

File: front/package-lock.json [hunk 2]
```diff
@@ -5,7 +5,7 @@
   "packages": {
     "": {
       "dependencies": {
-        "@dust-tt/sparkle": "0.1.71-henry",
+        "@dust-tt/sparkle": "0.1.71",
```
Comment (spolu) [inline front/package-lock.json:8]: looks not rebased on main?

**Review by spolu — DISMISSED** [review]
LGTM

Smal naming nit

File: front/pages/w/[wId]/builder/assistants/[aId]/index.tsx (hunk 1 above)
Comment (fontanierh) [reply front/pages/w/[wId]/builder/assistants/[aId]/index.tsx]: fixed

File: front/package-lock.json (hunk 2 above)
Comment (fontanierh) [reply front/package-lock.json:8]: It is actually, in previous PR I forgot to npm install after switching my sparkle (this is package lock)

Comment (spolu) [reply front/package-lock.json:8]: 👍

**Review by spolu — APPROVED** [review]

**Pull request merged**